src/site/content/en/blog/why-coop-coep/index.md

```diff
@@ -0,0 +1,236 @@
+If a cross origin resource supports [Cross Origin Resource Sharing
+(CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS), you may use the
+[`crossorigin`
+attribute](https://developer.mozilla.org/docs/Web/HTML/Attributes/crossorigin)
+to load it to your web page without being blocked by COEP.
+
+```html
+<img src="https://third-party.example.com/image.jpg" crossorigin>
+```
+
+For example, if this image resource is served with CORS headers, use  the
+`crossorigin` attribute so that the request to fetch the resource will use [CORS
+mode](https://developer.mozilla.org/docs/Web/API/Request/mode). This also
+prevents the image from being loaded unless it sets CORS headers.
+
+Similarly, you may fetch cross origin data through the `fetch()` method, which
+doesn't require special handling as long as the server responds with [the right
+HTTP
+headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#The_HTTP_response_headers).
+
+### Cross Origin Resource Policy {: #corp }
+[Cross Origin Resource Policy
+(CORP)](https://developer.mozilla.org/docs/Web/HTTP/Cross-Origin_Resource_Policy_%28CORP%29)
+was originally introduced as an opt-in to protect your resources from being
+loaded by another origin. In the context of COEP, CORP can specify the resource
+owner's policy for who can load a resource.
+
+```http
+Cross-Origin-Resource-Policy: same-site
+Cross-Origin-Resource-Policy: same-origin
+Cross-Origin-Resource-Policy: cross-origin
+```
+
+Resources that are marked `same-site` can only be loaded from the same site,
+`same-origin` from the same origin. [`cross-origin` which was added to the CORP
+spec along with COEP](https://mikewest.github.io/corpp/#integration-fetch)
+declares that a resource can be loaded by any website.
+
+{% Aside %}
+Once you add the COEP header, you won't be able to bypass the restriction by
+using service workers. If the document is protected by a COEP header, the policy
+is respected before the response enters the document process, or before it
+enters the service worker that is controlling the document.
+{% endAside %}
+
+### Cross Origin Opener Policy {: #coop }
+[COOP (Cross Origin Opener
+Policy)](https://github.com/whatwg/html/pull/5334/files) allows you to ensure
+that a top-level window is isolated from other documents by putting them in a
+different browsing context group, so that they cannot directly interact with the
+top-level window. For example, if a document with COOP opens a pop-up, its
+`window.opener` will be `null`. Also, the `.closed` property of the opener's
+reference to it will return `true`.
+
+![COOP](coop1.png)
+
+The COOP header takes three values:
+
+```http
+Cross-Origin-Opener-Policy: same-origin
+Cross-Origin-Opener-Policy: same-origin-allow-popups
+Cross-Origin-Opener-Policy: unsafe-none
+```
+
+Documents that are marked `same-origin` can share the same browsing context
+group with same-origin documents that are also explicitly marked `same-origin`.
```

Comment:
Copy what I did for `Cross-Origin-Resource-Policy`.

Reply:
Done